### ISSUE TITLE
Implementation in Pytorch

### ISSUE THREAD

**Issue by ierosodin** [Oushesh/SegCaps-TF_Version#1]
I'm wondering if there is any implementation in Pytorch version?

**Comment by Oushesh** [comment]
At the time of implementation I did not know pytorch. Check on internet and let me know. if not I will reimplement using pytorch

Thanks

**Comment by ierosodin** [comment]
Yes, I check some web and cannot find any other implementation in Pytorch.
I also try to implement it according to the paper, but maybe I don't really understand about SegCaps, it pop out OOM! (in same dataset).
So I'm finding other implementation in Pytorch to help me clarify my concept.
Thanks
